db/migrations/20230104042228_create_plantsimg_table.sql

```diff
@@ -0,0 +1,11 @@
+-- migrate:up
+CREATE TABLE plantsimg (
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  imgurl VARCHAR(200) NOT NULL,
```

Comment:
```suggestion
  img_url VARCHAR(200) NOT NULL,
```
url은 때에 따라 200이 넘어갈 수도 있을 것 같습니다.